examples/gnrc_networking/README.md

```diff
@@ -96,10 +94,7 @@ Now, on the RIOT side, send a UDP packet using:
 
 You should see `testmessage` appear in netcat. Instead of using netcat,
 you can of course write your own software, but you may have to bind the
-socket to a specific interface (tap0 in this case). For an example that
-shows how to do so, see [here][tap-socket]
-
-[tap-socket]: (https://gist.github.com/backenklee/dad5e80b764b3b3d0d3e).
+socket to a specific interface (tap0 in this case). To do so, have a look at [setting the `SO_BINDTODEVICE` option using `setsocketopt()`](https://stackoverflow.com/questions/14478167/bind-socket-to-network-interface#14478657).
```

Comment:
Maybe just put the code here directly instead?

Reply:
I thought about that too at first, but since it's more of an add-on that has nothing to do with the RIOT part itself, I decided against it because adding a code snippet would draw too much attention to what is essentially a tooltip (imo) and this might cause confusion.

Reply:
Ok